src/04-modality-specific-files/10-eye-tracking.md

```diff
@@ -0,0 +1,187 @@
+# Eye-tracking including gaze position and pupil size
+
+Support for eye-tracking dataset developed as a [BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals). Please see [Citing BIDS](../01-introduction.md#citing-bids) on how to appropriately credit this extension when referring to it in the context of the academic literature.
+
+## Terminology and conventions
+
+Eye-tracking-BIDS is fully consistent with the BIDS specification as a whole. However, BIDS was initially developed in the context of MRI, so some terminology may be unfamiliar to researchers from other fields. This section adds clarifications to [Common Principles - Definitions](../02-common-principles.html) for the eye-tracking context.
```

Comment:
Could we use something like EyeTraking, to keep the dash just between modality and BIDS?

```suggestion
EyeTracking-BIDS is fully consistent with the BIDS specification as a whole. However, BIDS was initially developed in the context of MRI, so some terminology may be unfamiliar to researchers from other fields. This section adds clarifications to [Common Principles - Definitions](../02-common-principles.html) for the eye-tracking context.
```

Reply:
yes